src/components/IIIFPlayerWrapper.js

```diff
@@ -28,6 +29,17 @@ export default function IIIFPlayerWrapper({
     }
   }, []);
 
+  React.useEffect(() => {
+    if (manifest) {
+      //Parse manifest to see if auto-advance behavior present at manifest level then save into state
+      const manifestParsed = parseManifest(manifest);
+      const autoAdvanceBehavior = manifestParsed.getProperty("behavior")?.includes("auto-advance");
```

Comment:
I think it would be good to have all the `manifesto.js` parsing code in the `iiif-parser` module.
Maybe we could move this into a new function in the parser? What do you think?